format/ecs.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-param-reassign */
+const { format } = require('logform');
+
+/*
+ * function ecs (info)
+ * Aligns the base info object fields to the ECS schema
+ */
+module.exports = format((info) => {
+  if (info.timestamp) {
+    info['@timestamp'] = info.timestamp;
+    delete info.timestamp;
+  }
+
+  if (info.level) {
+    info.log = { level: info.level, ...info.log };
+    delete info.level;
+  }
+
+  if (info.stack) {
+    info.error = { stack_trace: info.stack, ...info.error };
+    delete info.stack;
+  }
+
+  if (Object.keys(info.strigo).length === 0) {
```

Comment:
Yes it should, but I've been lazy.

Reply:
I wouldnt want to bind `lodash` to this

Reply:
> What's `STRIGO_BASE`?

it's the key name under which all strigo related context goes.

Reply:
> And we're sure it will be an object always? Can it be null?
pretty sure, unless I misunderstood how `metadata` formatter works. can be null if metadata going to be removed.

Reply:
> I wouldnt want to bind `lodash` to this just for this simple check.

That's why I said _if_ we use one already :smirk:

Reply:
No longer relevant.